components/autocomplete/events.md

```diff
@@ -228,6 +230,75 @@ When using `OnRead`, make sure to set `TItem` and `TValue`.
 }
 ````
 
+## OnOpen
+
+The `OnOpen` event fires before the AutoComplete popup renders. 
+
+The event handler receives as an argument an `AutoCompleteOpenEventArgs` object that contains:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Property | Description |
+| --- | --- |
+| `IsCancelled` | Set the `IsCancelled` property to `true` to cancel the opening of the popup. |
+
+````CSHTML
+<TelerikAutoComplete Data="@Suggestions"
+                     @bind-Value="@AutoCompleteValue"
+                     OnOpen="OnOpenEventHandler" />
+
+@code {
+    private string AutoCompleteValue { get; set; }
+
+    private void OnOpenEventHandler(AutoCompleteOpenEventArgs args)
+    {
+        //set the IsCancelled to true to cancel the opening of the popup.
+        args.IsCancelled = false;
```

Comment:
This syntax may imply that the developer needs to set `args.IsCancelled` at all times. How about something like:

```suggestion
        //Set IsCancelled to true to cancel the opening of the popup.
        //args.IsCancelled = true;
```